content-src/lib/getRelativeTime.js

```diff
@@ -12,13 +12,13 @@ module.exports = function getRelativeTime(t) {
   const h = Math.floor((diff % TIME.d) / TIME.h);
   const d = Math.floor(diff / TIME.d);
   if (diff < TIME.m) {
-    return "<1m";
+    return {timestampID: "less_than_minute_label"};
```

Comment:
would you mind adding a jsdoc to this function just to describe the returned object now since it's a little more complicated?